armi/reactor/converters/uniformMesh.py

```diff
@@ -307,13 +312,13 @@ def applyStateToOriginal(self):
         runLog.extra(
             f"Parameter remapping time: {completeEndTime-completeStartTime} seconds"
         )
-        self.reset()
```

Comment:
There is a `meshConverter.reset()` call after `applyStateToOriginal` is called here:

https://github.com/terrapower/armi/blob/9d6a405f10a0084b2fea8721f2b6d08c16a0b7a8/armi/physics/neutronics/globalFlux/globalFluxInterface.py#L485

The same pattern exists in the one place I could find `applyStateToOriginal` in internal code.

You could make the argument that it's better to keep the one that's within `applyStateToOriginal` because then we wouldn't ever forget to call it. But I don't see why it's absolutely necessary to run `reset()` immediately after `applyStateToOriginal`. They seem like independent actions, so I think it's okay to keep them separate.